.github/workflows/publish.yml

```diff
@@ -0,0 +1,47 @@
+
+name: Publish to PyPI
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  packages:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+
+    - name: Get tags
+      run: git fetch --depth=1 origin +refs/tags/*:refs/tags/*
+      shell: bash
+
+    - name: Install build tools
+      run: |
+        python -m pip install --upgrade pip wheel setuptools setuptools_scm build twine
+      shell: bash
+
+    - name: Build binary wheel
+      run: python -m build --sdist --wheel . --outdir dist
+
+    - name: CheckFiles
+      run: |
+        ls dist
+      shell: bash
+
+    - name: Test wheels
+      run: |
+        cd dist && python -m pip install ioos_qc*.whl
+        python -m twine check *
+      shell: bash
+
+    # - name: Publish a Python distribution to PyPI
+    #   uses: pypa/gh-action-pypi-publish@master
+    #   with:
+    #     user: __token__
+    #     password: ${{ secrets.PYPI_PASSWORD }}
```

Comment:
Someone with a PyPI account and onwership to ioos_qc should create the token.
Actually is ioos_qc on PyPI?

Reply:
https://pypi.org/project/ioos-qc/

I will add the secret

Reply:
@mwengren can you make me a repository owner so I may add secrets? Alternatively I can send you the token to add via other secure methods, lmk!

Reply:
@kwilcox Sorry, sometimes GitHub traffic gets skipped over (by me, by accident).  I just gave you permissions.  LMK when this is working and I will change you back because we need to keep Admin privileges as restricted as possible.

Reply:
@kwilcox Are we good with this now?  Will reset permissions if so.

Reply:
Yes, i added the key, all set